### PR TITLE
ci: add workflows for next branch

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

As in title. See more [here](https://github.com/asyncapi/asyncapi-react/pull/320#issuecomment-828976524).

### CODE REVIEW

File: .github/workflows/release-wc-next.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# This action is a copy of `release-wc-and-playground` workflow for the purpose of the` next` branch only. 
+name: Release Web-component
+
+on: 
+  release:
+    branches:
+      - next
```
Comment (derberg) [inline .github/workflows/release-wc-next.yml:7]: is there such a thing as a release on a given branch only?

Comment (magicmatatjahu) [reply .github/workflows/release-wc-next.yml]: It doesn't work 😅 I changed to `${{ github.event.release.target_commitish }}` in refs.

File: .github/workflows/release-wc-and-playground.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v2
         with:
-          ref: master
+          ref: ${{ github.event.release.target_commitish }}
```
Comment (derberg) [inline .github/workflows/release-wc-and-playground.yml:17]: just curious what value is it? can you add a comment?

Comment (magicmatatjahu) [reply .github/workflows/release-wc-and-playground.yml:18]: Added

**Review by derberg — COMMENTED** [review]
I think you need to do something more to webcomponent workflow. Look at https://www.npmjs.com/package/semantic-release and notice **Published** column. We do not want it to be published as `latest` when released from `next` but as `next`.

We do not use semantic-release here so something has to be changed I think to avoid default

**Comment by magicmatatjahu** [comment]
@derberg You have:

```yaml
      - name: Checkout repository
        uses: actions/checkout@v2
        with:
          # target branch of release. More info https://docs.github.com/en/rest/reference/repos#releases
          ref: ${{ github.event.release.target_commitish }}
      - name: Setup Node.js
        uses: actions/setup-node@v1
        with:
          node-version: 14
      - name: Install latest version of library in the web-component
        run: VERSION=${{github.event.release.tag_name}} npm run install:reactcomp
        working-directory: ./web-component
      - name: Install dependencies
        run: npm install
      - name: Bump version in package.json
        # There is no need to substract "v" from the tag as version script handles it
        # When adding "bump:version" script in package.json, make sure no tags are added by default (--no-git-tag-version) as they are already added by release workflow
        # When adding "bump:version" script in package.json, make sure --allow-same-version is set in case someone forgot and updated package.json manually and we want to avoide this action to fail and raise confusion
        run: VERSION=${{github.event.release.tag_name}} npm run bump:webcomp:version
      - name: Release web-component to NPM
        uses: JS-DevTools/npm-publish@v1
        with:
          token: ${{ secrets.NPM_TOKEN }}
          package: ./web-component/package.json
          access: public
```

So `Bump version in package.json` should handle problem, yes? :)

**Comment by derberg** [comment]
@magicmatatjahu let me say it differently.

We need to make sure that once user runs `npm i @asyncapi/web-component` then `0.23.0` version is installed and to get next they need to do `npm i @asyncapi/web-component@1.0.0-next.1` so in npm **1.0.0-next.1** is not marked as **latest**

**Comment by magicmatatjahu** [comment]
@derberg Changed :)

**Comment by derberg** [comment]
@magicmatatjahu I tried to establish a tunnel connection with Bieruń using putty but I failed, sorry 😄 Maybe easier will be when you actually push your commit to remote 😛

**Review by derberg — APPROVED** [review]
lgtm, well done!

**Pull request merged**